CHANGELOG.md

```diff
@@ -3,6 +3,48 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.14.0
+
+_Jul 21, 2022_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 🧪 Export previously accessible components
```

Comment:
Not sure to understand that one

Reply:
There are 2 new exported grid components. I'm open to suggestions.
We can also remove this line and just have bugfixes and docs improvements but because of this it is a minor version bump.

Reply:
The components are `GridBooleanCell` and which one ?

Reply:
Ou sorry just this one. I misread another PR title. I'll add it as a highlight.